omnibot/routes/api.py

```diff
@@ -200,7 +200,10 @@ def slack_slash_command():
         )
         logger.error(
             msg,
-            extra={'team': team.team_id},
+            extra=merge_logging_context(
```

Comment:
can we use unpacking here instead? 
```
{ **{'expected_team_id': team.team_id }, **bot.logging_context }
```

Reply:
I'd really prefer not to unpack in every spot. it's easier to read with the function, and it's a lot easier to have things be written correctly if we just need to call a function with dicts that returns a dict.